proxyclient/m1n1/fw/afk/rbep.py

```diff
@@ -50,19 +52,24 @@ class AFKEP_Shutdown(AFKEPMessage):
 class AFKEP_Shutdown_Ack(AFKEPMessage):
     TYPE = 63, 48, Constant(0xc1)
 
-
-class AFKError(Exception):
-    pass
+# TODO no one else but AOP uses AFKRingBuf right?
```

Comment:
DCP uses AFKRingBuf on some endpoints via `EPICEndpoint`. Used in experiments/dcpext_iboot.py

Reply:
New commits don't break DCP experiments. DCP/AOP AFKRingbuf are too different to share (e.g. different EPIC header, different ring size, AOP also writes ptrs to DART and etc), everything else is fine though. I think we can move afk to soc/apple/ and just have fn pointers for the rbep r/w calls